plutus-playground-client/src/Chain/BlockchainExploration.purs

```diff
@@ -215,7 +215,7 @@ balanceView Remainder =
 
 valueView :: forall p i. Tuple CurrencySymbol Int -> HTML p i
 valueView (Tuple (CurrencySymbol sym) balance) =
-  amountView (show sym) balance
+  amountView ("λ" <> show sym) balance
```

Comment:
what's with all these lambdas all over the place?

Reply:
I needed something to indicate, "this is a currency". It seemed to me to be the nicest Greek symbol, but I'm open to suggestions. 

@michaelpj @j-mueller Is there a standard symbol for "this is some currency in our model" perhaps?

Reply:
Can't we suffix with the currency symbol? e.g. `10 EUR`, `50 USD`?

Reply:
I've got a couple enhancements to do in this direction, so I think I'll roll that into a different PR.